assets/css/main.css

```diff
@@ -1,624 +1,693 @@
+/*
+Red/Teal
+rgba(204, 40, 41, 1), #4ba58c
+*/
+/*
+Grey-Blue
```

Comment:
I'd remove this comment block or fill out all the colors and give a short description of when you should use each of them